controllers/machinedeployment_controller.go

```diff
@@ -117,6 +117,9 @@ func (r *MachineDeploymentReconciler) reconcile(ctx context.Context, d *clusterv
 		return ctrl.Result{}, errors.Errorf("failed validation on MachineDeployment %q label selector, cannot match Machine template labels", d.Name)
 	}
 
+	// Copy label selector to its status counterpart in string format.
```

Comment:
Should we add this is necessary for the scale subresource to work?

Reply:
Sure thing

Reply:
Added